### PR TITLE
[ACR] Prepare release

### PR DESCRIPTION
**Pull request by YalinLi0312**

### CODE REVIEW

**Review by mccoyp — COMMENTED** [review]
Should this release be version 1.1.1 since it looks like there's just a bug fix?

File: sdk/containerregistry/azure-containerregistry/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,10 @@
 # Release History
 
-## 1.1.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.2.0 (2023-07-11)
```
Comment (kristapratico) [inline sdk/containerregistry/azure-containerregistry/CHANGELOG.md:3]: Should this only be a patch bump?

**Comment by YalinLi0312** [comment]
Have talk with @annatisch, since there has api updates, it should be version `1.2.0`.

**Comment by mccoyp** [comment]
> Have talked with @annatisch, since there has api updates, it should be version `1.2.0`.

I would suggest documenting the API change(s) in `Features Added` or `Other Changes`, just to make that more clear to users. Makes sense to do a minor bump in that case though 🙂

**Comment by kristapratico** [comment]
> Have talked with @annatisch, since there has api updates, it should be version `1.2.0`.

Got it, it looks like the version in _version.py needs update as well.

**Review by kristapratico — APPROVED** [review]

**Comment by YalinLi0312** [comment]
> > Have talked with @annatisch, since there has api updates, it should be version `1.2.0`.
> 
> I would suggest documenting the API change(s) in `Features Added` or `Other Changes`, just to make that more clear to users. Makes sense to do a minor bump in that case though 🙂

This is a good point, will update!

**Review by mccoyp — APPROVED** [review]

**Pull request merged**